prometheus-hanadb_exporter.spec

```diff
@@ -64,9 +65,9 @@ mkdir -p %{buildroot}%{oldsyscondir}/hanadb_exporter
 mkdir -p %{buildroot}%{_sysconfdir}/hanadb_exporter
 install -D -m 644 daemon/hanadb_exporter@.service %{buildroot}%{_unitdir}/hanadb_exporter@.service
 
-install -D -m 0644 config.json.example %{buildroot}%{_docdir}/hanadb_exporter/config.json.example
-install -D -m 0644 metrics.json %{buildroot}%{_docdir}/hanadb_exporter/metrics.json
-install -D -m 0644 logging_config.ini %{buildroot}%{_docdir}/hanadb_exporter/logging_config.ini
+install -D -m 0644 config.json.example %{buildroot}%{_docdir}/%{name}/config.json.example
+install -D -m 0644 metrics.json %{buildroot}%{_docdir}/%{name}/metrics.json
+install -D -m 0644 logging_config.ini %{buildroot}%{_docdir}/%{name}/logging_config.ini
```

Comment:
are we sure this doesn't break anything?
As far as I'm concerned, we could just use the short name here, without the `prometheus-` prefix, so that we are also consistent with how the executable is installed.

Reply:
withouth the `%(name}` macro the pkg doesn't build

Reply:
you can check the build status here: https://build.opensuse.org/package/show/server:monitoring/prometheus-hanadb_exporter

Reply:
oh nevermind, I thought those were the targets, not the sources!